conf/machine/include/imx8mn-evk.inc

```diff
@@ -0,0 +1,52 @@
+MACHINEOVERRIDES =. "mx8:mx8m:mx8mn:"
+
+require conf/machine/include/imx-base.inc
+require conf/machine/include/tune-cortexa53.inc
+
+MACHINE_FEATURES += "wifi bluetooth optee jailhouse bcm43455 bcm4356"
+
+# NXP BSP can consume proprietary Broadcom driver
+MACHINE_FEATURES_append_use-nxp-bsp = " bcm4359"
+
+KERNEL_DEVICETREE_BASENAME ?= "${MACHINE}"
```

Comment:
Should `KERNEL_DEVICETREE_BASENAME` be left undefined in order to enforce target machine configurations to set it explicitly?

If not - then `UBOOT_CONFIG_BASENAME` should also receive default value then to maintain consistency.